utils/obsproc/util.h

```diff
@@ -219,19 +221,33 @@ namespace gdasapp {
           oops::Log::test() << checksum(datetime_, "datetime") << std::endl;
         }
 
-       // Swap the date
-       void reDate(int64_t minDAwindow, int64_t maxDAwindow, int windowcyc) {
+       // Convert JulianDate and second
+       void JulianDateTime(const std::string & str, int64_t & DAwindow) {
```

Comment:
fine for now, but this should be a general utility if that doesn't already exist somewhere else.